### ISSUE TITLE
Improve contrast ratio for greyed out buttons. 

### ISSUE THREAD

**Issue by martinfrances107** [jsdrupal/drupal-admin-ui#264]
With regard to the content page.

The operations of the actions pulldown select is "greyed out" until items are selected.

BUT chrome's lighthouse fails a contrast ration test as part of the accessibility checks 

I am splitting this off the #178 so that it can be seen by a different audience 

It needs a designers touch because some of the solutions I see need a change to the overall look and feel of the site.

**Comment by martinfrances107** [comment]
Here is a screenshot showing "Actions"  low contrast ratio.

![actionsfailscontrastratiotest](https://user-images.githubusercontent.com/1259969/44219612-6eae6000-a174-11e8-990a-6a7a361f4cb5.png)

**Comment by martinfrances107** [comment]
I think there are other ways of solving this problem...

There are other issues talking about theme management .. and graphic design which are a natural fit for contrast ratio argument.
